src/components/Infra/InfraEnvWizard.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react'
+import { InfraWizard } from 'openshift-assisted-ui-lib';
+import { useK8sModel, k8sCreate, history, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk/api';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { load } from 'js-yaml';
+
+import { InfraEnvKind } from '../../kind';
+
+import 'openshift-assisted-ui-lib/dist/index.css';
```

Comment:
This is not needed, already included in the `styles.scss`

Reply:
changed the import to `../styles.scss`;